app/js/openmrs.js

```diff
@@ -79,9 +78,16 @@ angular.module('openmrs.resources', [ 'ngResource' ])
 					});
 				},
 	
-				listFull : function(resource, query) {
+				listFull : function(resource) {
 					openmrsApi.add(resource);
-					query = angular.extend(query, {mode: 'v=full'});
+					var query = {v: 'full'};
```

Comment:
You are always creating a new query. My point was that you extend a query if it is passed or create a new one if it is null.